packages/checkbox/src/react/index.js

```diff
@@ -29,10 +30,22 @@ const styles = {
     )
 }
 
-const Checkbox = React.forwardRef((props, ref) => {
+const Checkbox = React.forwardRef((props, forwardedRef) => {
   const themeName = useTheme()
+
+  const ref = React.useRef()
+  React.useImperativeHandle(forwardedRef, () => ref.current)
+
   const square = React.createRef()
 
+  React.useEffect(
+    function updateIndetermiateAttr() {
+      if (!ref.current) return
+      ref.current.indeterminate = props.indeterminate
```

Comment:
the `indeterminate` attribute has to use the node setter - it isn't an html attribute